hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/helpers/TestProtoConversionUtil.java

```diff
@@ -206,7 +233,7 @@ private Pair<Sample, GenericRecord> createInputOutputSampleWithRandomValues(Sche
     long primitiveFixedSignedLong = RANDOM.nextLong();
     boolean primitiveBoolean = RANDOM.nextBoolean();
     String primitiveString = randomString(10);
-    byte[] primitiveBytes = getUTF8Bytes(randomString(10));
+    byte[] primitiveBytes = randomString(10).getBytes();
```

Comment:
Similar here on unintended changes.  In OSS, we have explicitly enforced UTF-8, although `.getBytes()` implicitly uses `UTF-8` on UNIX-like systems.